src/tests/ngraph_helpers/snippets_ngraph_functions/src/precision_propagation_convertion_function.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "precision_propagation_convertion_function.hpp"
+#include <assert.h>
+#include <ngraph/opsets/opset1.hpp>
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+namespace {
+std::shared_ptr<ngraph::op::FakeQuantize> make_fake_quantize(
+    const Output<Node>& parent,
+    const ngraph::PartialShape& inputShape,
+    const element::Type inputType,
+    const std::vector<float>& fake_quantize_intervals) {
+    auto generate = [](const ov::element::Type precision,
+        const ngraph::Shape& shape,
+        const float initialValue,
+        const std::string& name) {
+            const auto size = ngraph::shape_size(shape);
+            std::vector<float> values(size);
+            for (auto i = 0; i < size; ++i) {
+                values[i] = static_cast<float>(initialValue + i);
+            }
+            auto constant = std::make_shared<ngraph::opset1::Constant>(precision, shape, values);
+            constant->set_friendly_name(name);
+            return constant;
+    };
+
+    const auto fakeQuantize = std::make_shared<ngraph::opset1::FakeQuantize>(
+        parent,
+        generate(inputType, {}, fake_quantize_intervals[0], "inputLow"),
+        generate(inputType, {}, fake_quantize_intervals[1], "inputHigh"),
+        generate(inputType, {}, fake_quantize_intervals[2], "outputLow"),
+        generate(inputType, {}, fake_quantize_intervals[3], "outputHigh"),
+        256ul);
+    fakeQuantize->set_friendly_name("fakeQuantize");
+
+    return fakeQuantize;
+}
+} // namespace
+
+std::shared_ptr<ov::Model> PrecisionPropagationConvertionFunction::get(
+    const std::vector<ov::PartialShape>& input_shapes,
+    const element::Type input_type,
+    const std::vector<float>& fake_quantize_intervals) {
+    assert(2ull == input_shapes.size());
+    assert(4ull == fake_quantize_intervals.size());
+    const auto parameter1 = std::make_shared<ngraph::opset1::Parameter>(input_type, input_shapes[0]);
+    parameter1->set_friendly_name("parameter1");
+
+    const auto parameter2 = std::make_shared<ngraph::opset1::Parameter>(input_type, input_shapes[1]);
+    parameter2->set_friendly_name("parameter2");
+
+    std::shared_ptr<Node> parent = std::make_shared<ngraph::opset1::MaxPool>(
```

Comment:
Why do we need MaxPool after the Parameter?
To avoid this and further similar question in future, please add a brief description to your ngaph snippets function, as it's done for all other functions [example](https://github.com/openvinotoolkit/openvino/blob/6525dd47276939f87c9479f5e16ba0d9f036d1b7/src/tests/ngraph_helpers/snippets_ngraph_functions/include/subgraph_customizable.hpp#L21-L28)

Reply:
`MaxPool` was removed. The function is single FQ function.